lisa/tools/sysctl.py

```diff
@@ -13,5 +13,18 @@ def command(self) -> str:
     def can_install(self) -> bool:
         return False
 
-    def write(self, varivable: str, value: str) -> None:
-        self.run(f"-w {varivable}={value}")
+    def write(self, variable: str, value: str) -> None:
+        self.run(f"-w {variable}={value}", force_run=True, sudo=True)
+
+    def get(self, variable: str, force_run: bool = True) -> str:
+        result = self.run(
+            variable,
+            force_run=force_run,
+            sudo=True,
+            expected_exit_code=0,
+            expected_exit_code_failure_message=f"fail to get {self.command} {variable}",
+        )
+        # vm.oom_dump_tasks = 1
+        value = result.stdout.split("=")[-1].strip()
+        # return 1
```

Comment:
this comment is unclear and misleading. The return line is clear itself. And "return 1" is from above example, but no explanation here. So, it's confusing a little.